codestyle_checks.sh

```diff
@@ -4,4 +4,7 @@ echo "Running C++ code checks"
 cat utils/cpplint_files.txt | xargs python utils/cpplint.py --linelength=120 || exit 1
 
 echo "Running Python code checks"
-pep8 --first --max-line-length=99 python/artm/artm_model.py || exit 1
+pep8 --first --max-line-length=99 python/artm/model.py || exit 1
```

Comment:
Recommended code will check library.py and messages_pb2.py files. First one will give errors, and second one is auto-generated. Conditions inside the loop will make the script too complex.